github/repos_contents.go

```diff
@@ -72,7 +73,8 @@ func (r *RepositoryContent) Decode() ([]byte, error) {
 	return o, nil
 }
 
-// GetReadme gets the Readme file for the repository.
+// GetReadme gets the Readme file for the repository, which is stored as base64.
```

Comment:
I'm thinking what could be said safely is that it's retrieving encoded content. For instance: `// GetReadme gets the encoded readme file for the repository.`

Or was part of the point is you don't want to assume that it will remain encoded?